src/main/java/com/construction_worker_forum_back/service/UserService.java

```diff
@@ -17,25 +22,28 @@
 import org.springframework.security.core.userdetails.UsernameNotFoundException;
 import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Service;
-import org.springframework.util.StringUtils;
 import org.springframework.web.multipart.MultipartFile;
 import org.springframework.web.server.ResponseStatusException;
 
 import javax.transaction.Transactional;
-import java.io.FileInputStream;
+import java.io.File;
 import java.io.IOException;
 import java.time.Instant;
-import java.util.Date;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 
 @Service
-@AllArgsConstructor
```

Comment:
Why get rid of AllArgsConstructor ?

Reply:
We have value annotation on field which makes it useless. Its better to use RequiredArgsConstructor annotation